locust-metrics/app.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps.redskyops.dev/v1alpha1
+kind: Application
+metadata:
+  name: votingapp
+
+# The resources identify where the manifests representing the application live.
+# This is roughly the same as a Kustomize resources block, so you can use files,
+# directories, URLs, Git repos, etc.
+resources:
+- https://github.com/thestormforge/examples/voting-webapp/application
+
+# This section configures the objectives of optimization
+# We want to minimize both cost and latency
+objectives:
+- name: cost-gcp # Also "cost", "cost-aws", "cpu", ...
+- name: p95-latency
+  latency: p95
```

Comment:
Technically you don't need both of these: we [strip the word "latency"](https://github.com/thestormforge/optimize-controller/blob/31298dbee2ac2a1fba810e619832b0935dd123df/api/apps/v1alpha1/application_defaults.go#L139) from the name when trying to [decide what the default value](https://github.com/thestormforge/optimize-controller/blob/31298dbee2ac2a1fba810e619832b0935dd123df/api/apps/v1alpha1/latency.go#L30) should be.

Reply:
ok let me remove it then.